tests/moe_ep/test_sm90_push_fp8_packaging.py

```
@@ -419,3 +509,96 @@ def test_sm90_push_weights_do_not_depend_on_trace_templates():
+@pytest.mark.parametrize("module_name", ["nvfp4_w4a8_gemm", "nvfp4_rs_gemm"])
+def test_sm90_push_nvfp4_uri_covers_sources_dependencies_and_cuda_flags(
+    module_name, monkeypatch
+):
+    from dataclasses import replace
+
+    from flashinfer.moe_ep.kernel_src.sm90.push_style_megamoe import shim
+
+    module = getattr(shim, module_name)
+    snapshot = module._capture_source_snapshot()
+    if module_name == "nvfp4_w4a8_gemm":
+        digest = lambda value: module._source_digest(value)
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `digest` as a `def`

(E731)